web/docs/auth/auth-hooks.md

```diff
@@ -520,6 +528,54 @@ export const onAfterLogin: OnAfterLoginHook = async ({
 
 Read more about the data the `onAfterLogin` hook receives in the [API Reference](#the-onafterlogin-hook).
 
+### Refreshing the OAuth access token
+
+If you are using OAuth, you can use the `onAfterSignup` or `onAfterLogin` hooks to refresh the OAuth access token if the provider supports it.
```

Comment:
Which of these hooks should they use for this and why?

Can we recommend/explain it?

Reply:
I've rewritten this section to be more clear, thanks for the nudge 👍